---
title: Other repositories
contributors: [Marion Shadbolt]
---

## Generalist repositories

[https://sharing.nih.gov/data-management-and-sharing-policy/sharing-scientific-data/generalist-repositories](https://sharing.nih.gov/data-management-and-sharing-policy/sharing-scientific-data/generalist-repositories)

[https://www.re3data.org/](https://www.re3data.org/)

## Tool/workflow repositories


### ToolFinder

[ToolFinder](https://australianbiocommons.github.io/2_tools.html) is a service that lists bioinformatics tools and where they are already installed on Australian Computational Infrastructure, such as Galaxy, NCI and Pawsey. Tools can be queried, sorted and filtered and publications and links to documentation are provided where available. Tools are also curated with EDAM ontologies and cross-referenced to ELIXIRâ€™s [bio.tools](https://bio.tools/) repository.


### WorkflowFinder

[WorkflowFinder](https://australianbiocommons.github.io/2_1_workflows.html) is a service that lists workflows that have been submitted to WorkflowHub by Australian BioCommons partners. Workflows are curated with EDAM ontologies, tags and links out to documentation. Where available on Galaxy there is a link to open the workflow.


### scRNA-tools

[scRNA-tools](https://www.scrna-tools.org/) is a registry for software to analysis single-cell RNA-seq data [(Zappia, Phipson, and Oshlack 2018)](https://www.zotero.org/google-docs/?v3UOD4). There are over 1300 tools in over 30 categories that have been added to date. Users can [nominate new tools](https://www.scrna-tools.org/submit) to be added through a webform. [Stats](https://www.scrna-tools.org/analysis) on the tools are presented and tools can be queried and sorted in a number of different ways, such as by the number of citations or how recently it was updated.


### WorkflowHub

WorkflowHub is a registry run by ELIXIR to describe, share and publish scientific computational workflows. Each entry links out to where a workflow has been stored, such as GitHub. Users can also add documentation, tags and attributions.


### Bio.tools

[Bio.tools](https://bio.tools/) is an ELIXIR resource that provides a searchable interface for scientific and technical information about bioinformatics tools, databases and services. Tools can be [submitted](https://biotools.readthedocs.io/en/latest/quickstart_guide.html) for inclusion by the developer or by other community members. Tools are given unique identifiers and curated with EDAM ontologies within the [biotoolsSchema](https://biotoolsschema.readthedocs.io/en/latest/). The registry is also [queryable via API](https://biotools.readthedocs.io/en/latest/api_usage_guide.html), allowing other tools to be built on top of it. Entries for each tool link out to where the tool can be downloaded, e.g. GitHub.


### BioCode

[BioCode](https://ngdc.cncb.ac.cn/biocode) is a registry run by the CNCB-NGDC to archive Bioinformatics software tools. It stores the software within its own database and can thus track the number of downloads and provides summary statistics. Registered users are also able to rate code for usability, efficiency and reliability. Publications can also be linked to the registered code.

## Protocol repositories

### Protocols exchange

[https://protocolexchange.researchsquare.com/](https://protocolexchange.researchsquare.com/)


### Protocols.io [https://www.protocols.io/](https://www.protocols.io/)

Protocol journals

[https://www.researchprotocols.org/](https://www.researchprotocols.org/)

[https://www.mdpi.com/journal/mps](https://www.mdpi.com/journal/mps)

[https://www.nature.com/nprot/](https://www.nature.com/nprot/) Nature protocols

[https://currentprotocols.onlinelibrary.wiley.com/](https://currentprotocols.onlinelibrary.wiley.com/)

[https://www.cell.com/star-protocols/home](https://www.cell.com/star-protocols/home)

[https://bio-protocol.org/default.aspx](https://bio-protocol.org/default.aspx)
